### PR TITLE
Palindrome checker responsive

### LINKED ISSUE #62: Make the existing Palindrome Checker responsive
The project is present at: `/public/Projects/VaibhavPalindromeCheck`

- On large screens everything looks fine.
- However, in mobile view the main content of the project is scrolled towards the right of the screen.
- Your task is to make the project responsive so that the input box and button remains in the center on smaller screens.

**intended file to be changed:**
- `/public/Projects/VaibhavPalindromeCheck/index.html`
- You may add a `.css` file inside `/public/Projects/VaibhavPalindromeCheck` if you want
- All other files should remain unchanged before commiting

### PR DESCRIPTION
**Pull request by LucianeFernandesRoque**
close #62 

# Description
We need add responsive design

---
## Type of change
Create file /assets/style.css 
remove styles on the page html

## Doc
'https://www.w3schools.com/'

--
## Images changes
Screenshot from 2022-10-28 16-39-09
Screenshot from 2022-10-28 16-38-05

### CODE REVIEW

File: public/Projects/VaibhavPalindromeCheck/index.html [hunk 1]
```diff
@@ -6,72 +6,73 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Palindrome-Checker</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css"
```
Comment (SandeepKrSuman) [inline public/Projects/VaibhavPalindromeCheck/index.html:9]: I guess keeping this bootstrap style was a good idea. 
The previous navbar and button style was looking better.
However if you can make them look better without bootstrap using CSS only then that will be nice as well. Keeping this upto you!

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]

**Comment by SandeepKrSuman** [comment]
Screenshot (671)

This definitely does not look good on larger screens. The previous one was better.

**Comment by SandeepKrSuman** [comment]
Also for mobile-view more can be done.
Screenshot (672)

**Review by SandeepKrSuman — APPROVED** [review]
Looks better than previous version 👍

**Comment by LucianeFernandesRoque** [comment]
> Looks better than previous version +1
You can do review of the my PR.
If you  have questions or change the code. Thank you for your time and tips

**Review by SandeepKrSuman — APPROVED** [review]

**Pull request merged**